x-pack/plugin/esql/qa/testFixtures/src/main/resources/math.csv-spec

```diff
@@ -432,14 +432,10 @@ powULIntOverrun
 ROW x = POW(9223372036854775808, 2)
 // end::powULOverrun[]
 ;
-// tag::powULOverrun-warning[]
-warning:Line 1:9: evaluation of [POW(9223372036854775808, 2)] failed, treating result as null. Only first 20 failures recorded.
-warning:Line 1:9: java.lang.ArithmeticException: long overflow
-// end::powULOverrun-warning[]
 
 // tag::powULOverrun-result[]
-x:long
-null
+x:double
```

Comment:
If this no longer overruns, we cannot use this in the documentation. Perhaps you need to delete the entire docs section regarding arithmetic overruns?

Reply:
Since you deleted the docs, you could also delete the tags here.